package.json

```diff
@@ -54,6 +54,7 @@
     "@govuk-one-login/di-ipv-cri-common-express": "6.3.0",
     "@govuk-one-login/frontend-analytics": "1.0.3",
     "@govuk-one-login/frontend-language-toggle": "1.1.0",
+    "@govuk-one-login/frontend-passthrough-headers": "^1.0.0",
```

Comment:
Non specific package versions are usually flagged during implementations, can we remove the caret here?

Reply:
I pushed the change